SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -82,15 +89,38 @@ Worker_RequestId USpatialWorkerConnection::SendCreateEntityRequest(TArray<FWorke
 	{
 		Data.Emplace(SpatialGDK::OwningComponentDataPtr(Component.schema_type), Component.component_id);
 	}
+
 	return Coordinator->SendCreateEntityRequest(MoveTemp(Data), Id, TOptional<uint32>(), SpanId);
 }
 
+Worker_RequestId USpatialWorkerConnection::SendCreateEntityRequest(TArray<FWorkerComponentData> Components, const Worker_EntityId* EntityId,
+																   SpatialGDK::FRetryData RetryData, const TOptional<Trace_SpanId>& SpanId)
+{
+	check(Coordinator.IsValid());
+	const TOptional<Worker_EntityId> Id = EntityId ? *EntityId : TOptional<Worker_EntityId>();
+	TArray<SpatialGDK::ComponentData> Data;
+	Data.Reserve(Components.Num());
+	for (auto& Component : Components)
+	{
+		Data.Emplace(SpatialGDK::OwningComponentDataPtr(Component.schema_type), Component.component_id);
+	}
+
+	return Coordinator->SendCreateEntityRequest(MoveTemp(Data), Id, RetryData, SpanId);
+}
+
 Worker_RequestId USpatialWorkerConnection::SendDeleteEntityRequest(Worker_EntityId EntityId, const TOptional<Trace_SpanId>& SpanId)
 {
 	check(Coordinator.IsValid());
 	return Coordinator->SendDeleteEntityRequest(EntityId, TOptional<uint32>(), SpanId);
 }
 
+Worker_RequestId USpatialWorkerConnection::SendDeleteEntityRequest(Worker_EntityId EntityId, SpatialGDK::FRetryData RetryData,
+																   const TOptional<Trace_SpanId>& SpanId)
+{
+	check(Coordinator.IsValid());
+	return Coordinator->SendDeleteEntityRequest(EntityId, SpatialGDK::RETRY_UNTIL_COMPLETE, SpanId);
```

Comment:
This should use the retry data in the argument

Reply:
ah, missed that. thanks!